torch/distributed/distributed_c10d.py

```diff
@@ -918,14 +927,14 @@ def is_backend_available(backend: str) -> bool:
 
 
 def is_initialized() -> bool:
-    """
-    Checking if the default process group has been initialized
-    """
+    """Check if the default process group has been initialized."""
     return GroupMember.WORLD is not None
 
 
 def is_torchelastic_launched() -> bool:
     """
+    Check process launch.
```

Comment:
Is this necessary to add? The snippet below is more descriptive.

Reply:
@H-Huang Yes, the multi-line doc string are required to have a one line summary, followed by space and then description. So for this changes and others similar changes where you have a similar comment, I tried to put one line summary from the long description. Without the changes, it gives following error:  
```
$ pydocstyle torch/distributed/distributed_c10d.py --count
torch/distributed/distributed_c10d.py:935 in public function `is_torchelastic_launched`:
        D205: 1 blank line required between summary line and description (found 0)
torch/distributed/distributed_c10d.py:935 in public function `is_torchelastic_launched`:
        D400: First line should end with a period (not '`')
torch/distributed/distributed_c10d.py:935 in public function `is_torchelastic_launched`:
        D401: First line should be in imperative mood (perhaps 'Check', not 'Checks')
3
```
So, I hope the changes are okay? Please let me know. Thanks!